test/src/test/java/jenkins/model/JenkinsBuildsAndWorkspacesDirectoriesTest.java

```diff
@@ -258,6 +292,15 @@ private boolean logWasFound(String searched) {
                 .anyMatch(record -> record.getMessage().contains(searched));
     }
 
+	private boolean logLevelWasFound(String searched, Level level) {
```

Comment:
`logWasFoundAtLevel` would probably better reflect what it does.
Also, I think `logWasFound` should delegate to it.